### PR TITLE
NativeScript Windows installer

### PR DESCRIPTION
**Pull request by NickIliev**
Article about the NativeScript Windows installer - merge after the installer is released

### CODE REVIEW

**Review by tsonevn — APPROVED** [review]

File: start/ns-setup-installer.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+position: 60
+slug: installer
+publish: false
+---
+
+# NativeScript CLI installer
+
+## Installation
+
+Install the latest version of NativeScript CLI and its missing prerequisites for Windows OS via executible installer.
```
Comment (tjvantoll) [inline start/ns-setup-installer.md]: via an executable installer.

File: start/ns-setup-installer.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+### Prerequisites
+
+The installation requires .NET Framework 4 and Windows 7 SP1 or later.
+
+In case .NET Framework 4 is missing the installer offers to download and install it.
+
+![](../img/start/installer-001.png "Initial screen")
+
+In case the OS prerequisite is not met a message pops up and the installer is shut down.
```
Comment (tjvantoll) [inline start/ns-setup-installer.md]: is not met, a message

File: start/ns-setup-installer.md [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+
+![](../img/start/installer-002.png "Initial screen")
+
+### Welcome Page
+
+![](../img/start/installer-003.png "Welcome screen")
+
+Welcome page describes what will be done by the installer. It also allows the user to review the GitHub repository and the license agreement of the product.
```
Comment (tjvantoll) [inline start/ns-setup-installer.md]: The welcome page

File: start/ns-setup-installer.md [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+
+Shows whether a supported JDK version is installed on the machine.
+
+![](../img/start/installer-009.png "JDK version")
+
+If no such version detected:
+
+- &quot;Supported JDK version not found&quot; message appears and an information about where it can be downloaded from and the &quot;Install&quot; button is disabled
+
+When supported JDK version is detected the Install button is enabled and the user can proceed. An additional setup action is shown in case JAVA\_HOME environment variable is not set or it points to a wrong location. If the variable is set and valid the additional setup action is not shown.
```
Comment (tjvantoll) [inline start/ns-setup-installer.md]: in case the `JAVA_HOME`

File: start/ns-setup-installer.md [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+
+If a version higher than the supported is detected – a warning message is shown.
+
+#### Android SDK
+
+Shows whether Android SDK is installed on the machine.
+
+![](../img/start/installer-015.png "Android SDK version")
+
+If Android SDK is not detected an information about what version will be installed and how much download and installation space it requires is shown.
```
Comment (tjvantoll) [inline start/ns-setup-installer.md]: If an Android SDK is not detected, information about

File: start/ns-setup-installer.md [hunk 6]
```diff
@@ -0,0 +1,112 @@
...
+
+Whether Android SDK is installed or not, the following setup actions are available:
```
Comment (tjvantoll) [inline start/ns-setup-installer.md]: the Android SDK

**Review by tjvantoll — COMMENTED** [review]
I added a couple of minor grammatical notes.

**Comment by tjvantoll** [comment]
Let’s make sure a link to the installer is included at the top of this article now also https://www.nativescript.org/docs/default-source/default-document-library/nativescript-cli-setup-wix-20170220.exe?sfvrsn=2

**Comment by NickIliev** [comment]
@tjvantoll Thanks for the revision - I am currently waiting for the official link which will be included at the top of the article.

**Pull request merged**